README.md

```diff
@@ -88,3 +88,13 @@ You should see output similar to the following:
     todo_app\tests\test_view_model.py .                                                                                                                                                                      [100%] 
 
     ============================ 1 passed, 2 deselected in 0.03s ============================ 
+
+## Provision in cloud using Ansible 
+
+Populate inventory list of IPs of managed nodes to provision:
+
+    ec2-user@ControlNode:/home/ec2-user $ vim .ansible/inventory.ini
+
+Command to provision a VM from an Ansible Control Node
+
+    ec2-user@ControlNode:/home/ec2-user $ ansible-playbook .ansible/playbook2.yaml -i .ansible/inventory.ini
```

Comment:
It would be good to extend this with your Docker commands as I had to work those all out!
````suggestion
    ec2-user@ControlNode:/home/ec2-user $ ansible-playbook .ansible/playbook2.yaml -i .ansible/inventory.ini
    
## Docker

You can build the two different environment containers with the appropriate command:
```bash
docker build -t leon --target development .
docker build -t leon:prod --target production .
```
And run with:
```bash
docker run -it --env-file .env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/opt/todoapp/todo_app leon

docker run -it --env-file .env -p 5100:8000 leon:prod
```
````